composer.json

```diff
@@ -10,7 +10,7 @@
         "php": ">=5.2.4"
     },
     "require-dev": {
-        "phpunit/phpunit": "3.7.*"
+        "phpunit/phpunit": "> 5.4.3"
```

Comment:
The PHPUnit requires `php-5.6` at least.

We have two approaches about using namespace PHPUnit:

- Using the PHPUnit `^4.8.36` version to support namespace PHPUnit.
- Drop `php-5.x` supports and upgrade the PHPUnit version to `^6.5` at least.